docs/concepts/time-and-windows-in-ksqldb-queries.md

```diff
@@ -391,9 +391,12 @@ For more information on joins, see
 
 ### Out-of-order events
 
-Frequently, events that belong to a window can arrive out-of-order, for example, over slow networks, 
-and a grace period may be required to ensure the events are accepted into the window. 
-ksqlDB enables configuring this behavior, for each of the window types. 
+Frequently, events that belong to a window can arrive out-of-order, for
+example, over slow networks, and a grace period may be required to ensure the
+events are accepted into the window. ksqlDB enables configuring this behavior
+for each of the window types and uses a default of 24 hours when no grace
+period is specified. Confluent recommends setting this value explicitly to fit
+your scenario. 
```

Comment:
Not sure if we want to include it, but we also plan to make `GRACE` mandatory in the future.

Reply:
Since it is related to a future release, I was thinking we can just keep the recommendation to specify it for now.